squad_client/manage.py

```diff
@@ -24,6 +25,7 @@ def main():
     parser.add_argument('--debug', action='store_true', help='display debug messages')
     parser.add_argument('--squad-host', help='SQUAD host, example: https://qa-reports.linaro.org')
     parser.add_argument('--squad-token', help='SQUAD authentication token')
+    parser.add_argument('--cache', default=3600, help='Cache API results for 3600s (or 1 hour). Disable it by passing 0 (zero)')
```

Comment:
I would say default should be 0. This is dumb cache and won't invalidate the results if they change on the remote end. It's good for debugging but not for production use.

Reply:
Yeah, I was questioning myself which should be the default. I guess we can enable caching by default once it gets smarter. Fixed it already